_includes/head.html

```diff
@@ -29,6 +29,11 @@
   {% if site.search_enabled != false %}
     <script type="text/javascript" src="{{ '/assets/js/vendor/lunr.min.js' | relative_url }}"></script>
   {% endif %}
+
+  {% if site.mermaid_enabled != false %}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mermaid/{{ site.mermaid.version }}/mermaid.min.js"></script>
```

Comment:
Is there a reason that we use this cloudflare CDN? On the [usage docs](https://mermaid-js.github.io/mermaid/#/n00b-gettingStarted) it seems to prefer jsdelivr. I actually personally have no strong opinion, just curious.

Reply:
No. That was done in #830. I agree that we should use their preferred. I've updated it accordingly.